modules/adding-to-a-project.adoc

```diff
@@ -6,18 +6,21 @@
 [id="adding-to-a-project_{context}"]
 = Adding to a project
 
-.Procedure
+You can add items to your project by using the *Add+* page in the *Developer* perspective.
+
+.Prerequisites
 
-. Select *Developer* from the context selector at the top of the web console
-navigation menu.
+* You have created a project.
+
+.Procedure
 
-. Click *+Add*
+. In the *Developer* perspective, navigate to the *+Add* page.
```

Comment:
I might be misunderstanding this, but should this say "Add+" instead of "+Add"? It appears as "Add+" everywhere else in this document.

Reply:
It should be *Add+*. Thanks for catching the others in the file. I will fix them all.

![image](https://github.com/openshift/openshift-docs/assets/25209155/15656e14-bc2d-443f-a6ce-a0bbf6c5ab4b)

Reply:
Actually, it's just the *Add* page; but that doesn't seem to be used in the docs. 

![image](https://github.com/openshift/openshift-docs/assets/25209155/bb16cdad-d49d-41ed-9fb1-10ca86447800)

Reply:
I'm a CLI kind of guy, so I don't really know what's on the web console, haha. Just so long as it's consistent! :slightly_smiling_face: